### DISCUSSION TITLE
Teensy 4.1: Using SPI1 instead of standard SPI?

### DISCUSSION THREAD

**Discussion by nemesis2049** [olikraus/u8g2#2350] [Q&A]
Hello guys,

first of all i want to thank Oli for his great work for this library!

I use the Teensy 4.1 with a Waveshare Display 128x128 in hardware-mode (HW). So, thats works really fine and fast. The PJRC Teensy Audioshield uses the pins 10,11,12  - on the other hand they are nessecary for the display communication. So i want to move from SPI to SPI1 in the higher pin section. As far as i know i can use the "2ND"-configs for that. CS, DC and Reset works fine - the problem is MOSI and SCK.

In my case i would like to switch from:

```
//MOSI @ Pin 11, SCK @ Pin13 --> works
U8G2_SSD1327_WS_128X128_F_4W_HW_SPI u8g2(U8G2_R0, /* cs=*/ 36, /* dc=*/ 34, /* reset=*/ 35); 
```
to

```
//MOSI @ Pin 26, SCK @ Pin27 --> works not
U8G2_SSD1327_WS_128X128_F_2ND_4W_HW_SPI u8g2(U8G2_R0, /* cs=*/ 36, /* dc=*/ 34, /* reset=*/ 35); 
```

Do anyone have a idea whats wrong here?
I started a couple of tests but couldnt solve that issue.

Thanks for helping!

**Comment by olikraus** [comment]
https://github.com/olikraus/u8g2/issues/1956

or better here: https://github.com/olikraus/u8g2/issues/2050
